pkg/workflows/wasm/host/standard_test.go

```diff
@@ -105,6 +117,24 @@ func TestStandardModeSwitch(t *testing.T) {
 	t.Run("successful mode switch", func(t *testing.T) {
 		mockExecutionHelper := NewMockExecutionHelper(t)
 		mockExecutionHelper.EXPECT().GetWorkflowExecutionID().Return("id")
+		// Node calls may occur on initialization depending on the language.
+		var donCall1 bool
+		var nodeCall bool
+		var donCall2 bool
+		mockExecutionHelper.EXPECT().GetNodeTime().RunAndReturn(func() time.Time {
+			if donCall1 {
+				nodeCall = true
+			}
+			return time.Now()
+		})
+		// We want to make sure time.Sleep() is called at least twice in DON mode and once in node Mode
+		mockExecutionHelper.EXPECT().GetDONTime(mock.Anything).RunAndReturn(func(ctx context.Context) (time.Time, error) {
+			donCall1 = true
```

Comment:
set this in an else, otherwise, if they only call in don mode after they will both be set.